website/docs/r/data_forwarding_rule.html.markdown

```diff
@@ -9,25 +9,61 @@ description: |-
 Provider to manage [Sumologic Data Forwarding Rule](https://help.sumologic.com/docs/manage/data-forwarding/amazon-s3-bucket/#forward-datato-s3)
 
 ## Example Usage
+
+For Partitions
 ```hcl
+resource "sumologic_partition" "test_partition" {
+  name               = "testing_rule_partitions"
+  routing_expression = "_sourcecategory=abc/Terraform"
+  is_compliant       = false
+  retention_period   = 30
+  analytics_tier     = "flex"
+}
+
 resource "sumologic_data_forwarding_rule" "example_data_forwarding_rule" {
-    index_id = "00000000024C6155"
-    destination_id = "00000000000732AA"
-    enabled = "true"
-    file_format = "test/{index}/{day}/{hour}/{minute}"
-    payload_schema = "builtInFields"
-    format = "json"
+  index_id = sumologic_partition.test_partition.id
+  destination_id = "00000000000732AA"
+  enabled = true
+  file_format = "test/{index}/{day}/{hour}/{minute}"
+  payload_schema = "builtInFields"
+  format = "json"
+}
+```
+For Scheduled Views
+```hcl
+resource "sumologic_scheduled_view" "failed_connections" {
+  index_name = "failed_connections"
+  query = "_sourceCategory=fire | count"
+  start_time = "2024-09-01T00:00:00Z"
+  retention_period = 1
+  lifecycle {
+    prevent_destroy = true
+    ignore_changes = [index_id]
+  }
+}
+
+resource "sumologic_data_forwarding_rule" "test_rule_sv" {
+  index_id       = sumologic_scheduled_view.failed_connections.index_id
+  destination_id = sumologic_data_forwarding_destination.test_destination.id
```

Comment:
What happens when user supplies both `IndexName` and `IndexId` ?

Reply:
`IndexId` will simply be ignored ... as `IndexId` is never taken into consideration in PUT/POST calls .. please check the API documentation [here](https://api.sumologic.com/docs/#operation/updateScheduledView)

Reply:
We are just using `IndexId` while creating the DF Rule... it is never updated in the PUT call, nor it is ever required to maintain the SV resource's state.

Reply:
>IndexId will simply be ignored .

This behavior should be documented in the terraform docs.

Reply:
Yes, we are documenting it in the SV page itself as it belongs to SV.

Reply:
Updated the docs in the [latest](https://github.com/SumoLogic/terraform-provider-sumologic/pull/691/commits/651ae5560867ff1dc5318edd273fe2ae42588b39) commit.